internal/cmd/measurement_schema/client.go

```diff
@@ -0,0 +1,241 @@
+	}
+
+	resp, _, err := c.BucketApi.GetBucketsExecute(req)
+	if err != nil {
+		return nil, fmt.Errorf("failed to find bucket %q: %w", params.BucketName, err)
+	}
+	buckets := resp.GetBuckets()
+	if len(buckets) == 0 {
+		return nil, fmt.Errorf("no bucket found with name %q", params.BucketName)
+	}
+
+	return &orgBucketID{OrgID: buckets[0].GetOrgID(), BucketID: buckets[0].GetId()}, nil
+}
+
+type CreateParams struct {
+	internal.OrgBucketParams
+	Name           string
+	Columns        []api.MeasurementSchemaColumn
+	ExtendedOutput bool
+}
+
+func (c Client) Create(ctx context.Context, params CreateParams) error {
+	ids, err := c.resolveOrgBucket(ctx, params.OrgBucketParams)
+	if err != nil {
+		return err
+	}
+
+	res, _, err := c.BucketSchemasApi.
+		CreateMeasurementSchema(ctx, ids.BucketID).
+		OrgID(ids.OrgID).
+		MeasurementSchemaCreateRequest(api.MeasurementSchemaCreateRequest{
+			Name:    params.Name,
+			Columns: params.Columns,
+		}).
+		Execute()
+
+	if err != nil {
+		return err
+	}
+
+	return c.printMeasurements(ids.BucketID, []api.MeasurementSchema{res}, params.ExtendedOutput)
+}
+
+type UpdateParams struct {
+	internal.OrgBucketParams
+	Name           string
+	ID             string
+	Columns        []api.MeasurementSchemaColumn
+	ExtendedOutput bool
+}
+
+func (c Client) Update(ctx context.Context, params UpdateParams) error {
+	ids, err := c.resolveOrgBucket(ctx, params.OrgBucketParams)
+	if err != nil {
+		return err
+	}
+
+	var id string
+	if params.ID == "" && params.Name == "" {
+		return errors.New("measurement id or name required")
+	} else if params.ID != "" {
+		id = params.ID
+	} else {
+		id, err = c.resolveMeasurement(ctx, *ids, params.Name)
+		if err != nil {
+			return err
+		}
+	}
+
+	res, _, err := c.BucketSchemasApi.
+		UpdateMeasurementSchema(ctx, ids.BucketID, id).
+		OrgID(ids.OrgID).
+		MeasurementSchemaUpdateRequest(api.MeasurementSchemaUpdateRequest{
+			Columns: params.Columns,
+		}).
+		Execute()
+
+	if err != nil {
+		return err
+	}
+
+	return c.printMeasurements(ids.BucketID, []api.MeasurementSchema{res}, params.ExtendedOutput)
+}
+
+type ListParams struct {
+	internal.OrgBucketParams
+	Name           string
+	ExtendedOutput bool
+}
+
+func (c Client) List(ctx context.Context, params ListParams) error {
+	ids, err := c.resolveOrgBucket(ctx, params.OrgBucketParams)
+	if err != nil {
+		return err
+	}
+
+	req := c.BucketSchemasApi.
+		GetMeasurementSchemas(ctx, ids.BucketID).
+		OrgID(ids.OrgID)
+
+	if params.Name != "" {
+		req = req.Name(params.Name)
+	}
+
+	res, _, err := req.Execute()
+	if err != nil {
+		return err
+	}
+	return c.printMeasurements(ids.BucketID, res.MeasurementSchemas, params.ExtendedOutput)
+}
+
+func (c Client) printMeasurements(bucketID string, m []api.MeasurementSchema, extended bool) error {
+	if len(m) == 0 {
+		return nil
+	}
+
+	if c.CLI.PrintAsJSON {
+		return c.CLI.PrintJSON(m)
+	}
+
+	var headers []string
+	if extended {
+		headers = []string{
```

Comment:
The existing commands in `influx` tend to use more human-readable column headers in table mode, for example: https://github.com/influxdata/influx-cli/pull/54/files#diff-425b59256130c1373f5df8900912229d394ed1ed4debba3814d7cc9c77d0ec40R277-R281

Should this command use the same style?

Reply:
Yes, and done 👍🏻

Reply:
I see the human-friendly headers in the helper functions, but it looks like these slices still need updated to match

Reply:
I've moved to using constants to reduce the chance of error